src/cli/init/templates.ts

```diff
@@ -17,7 +17,31 @@ import { InitOptions } from "../types";
 
 export const { dependencies, devDependencies, peerDependencies, scripts, version } = packageJSON;
 
-export function genPkgJSON(opts: InitOptions, pgkVerOverride?: string) {
+type peprPackageJSON = {
+  data: {
+    name: string;
+    version: string;
+    description: string;
+    keywords: string[];
+    engines: { node: string };
+    pepr: {
+      uuid: string;
+      onError: string;
```

Comment:
So we safeguard the user from typing anything, should this be an enum of [Errors](https://github.com/defenseunicorns/pepr/blob/dd866e5d0b6558c4720c5e82485bd62e53a6b366/src/lib/errors.ts#L4)?

Reply:
Yep!